languages/auto-tags-smart-uk.po

```diff
@@ -0,0 +1,142 @@
+# Translation file for Auto Tags Smart
+# Copyright (C) 2025 Maison da Silva
+# This file is distributed under the same license as the Auto Tags Smart package.
+msgid ""
+msgstr ""
+"Project-Id-Version: Auto Tags Smart 1.0\n"
+"Report-Msgid-Bugs-To: https://maisondasilva.com.br/\n"
+"POT-Creation-Date: 2025-07-21 12:28+0000\n"
+"PO-Revision-Date: 2025-07-24 09:03+0000\n"
+"Last-Translator: Eugene Fox <eugene@xfox111.net>\n"
+"Language-Team: Ukrainian (Ukraine)\n"
+"Language: uk\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
+
+#~ msgid "Auto Tags Smart"
+#~ msgstr "Розумні Авто-позначки"
+
+#~ msgid "Settings"
+#~ msgstr "Налаштування"

```

Comment:
I've noticed that you have removed these strings from your Portuguese locale. I'm not sure if this is intentional, so I left these strings commented out instead.

Without these, the plugin's name and "Settings" button are not localized in Plugins section.

<img width="828" height="92" alt="Image" src="https://github.com/user-attachments/assets/a1362f31-2eb1-4113-8ae0-6302f0acc842" />

<img width="668" height="207" alt="Image" src="https://github.com/user-attachments/assets/8374be59-8485-494c-a81f-4b65988d2d05" />

Same is valid for Russian locale as well.